i18n/en/docusaurus-plugin-content-docs/current/guides/how-to-minify-screenshots.mdx

```diff
@@ -0,0 +1,36 @@
+import Admonition from "@theme/Admonition";
+
+# How to Reduce Screenshot Size
+
+## Introduction {#preface}
+
+One of the key features of testplane is the ability to test layouts using screenshots. For this, you can take a screenshot during different states of a web page in your tests and compare it with a reference.
+
+An example of such a comparison:
+
+![Screenshot Comparison](/img/docs/guides/how-to-minify-screenshots.diff-in-screenshot.png)
+
+<Admonition type="info" title="For Beginners">
+**Where do reference screenshots come from?**
+
+You wrote a test using the testplane _assertView_ command. The _assertView_ command compares the actual screenshot with the reference screenshot. An actual screenshot is one that the test takes during its run. It reflects the current state of your web page. On the first run of your test, you don't yet have a reference screenshot. Therefore, the _assertView_ command will fail with a message that the reference screenshot is not found. At the same time, you will have an actual screenshot. If the actual screenshot reflects the state of your web page you expect (consider as reference), then you need to _accept_ the actual screenshot as the reference one. To do this, you need to run testplane with an additional _--update-refs_ option. Alternatively, you can use the GUI mode provided by the _html-reporter_ plugin. With it, you can view the run report in GUI mode, ensure the actual screenshots are as you expect, and accept them using the _Accept_ button.
+
+</Admonition>
+
+## Problem {#problem}
+
+If your project has many tests that use layout validation via screenshots, then your project will have many screenshots. Sometimes these screenshots are quite large and can take up a lot of space on the file system.
+
+If the project is actively developed, each pull request will generate its own test run report, containing hundreds (or thousands) of reference screenshots. The larger the screenshots, the larger your reports will be. This means they will take longer to download, occupy more disk space, and so on.
+
+## Solution: testplane-image-minifier {#solution}
+
+To reduce the space screenshots occupy on disk, add the [testplane-image-minifier][testplane-image-minifier] plugin to your project. This plugin supports 8 levels of lossless image compression.
+
+How to add and configure the plugin is detailed in the [plugin documentation][testplane-image-minifier].
+
+## Keywords {#keywords}
+
+-   testplane-image-minifier
+
+[testplane-image-minifier]: ../../plugins/testplane-image-minifier
```

Comment:
Did you recently renamed the repo?
https://github.com/gemini-testing/testplane-image-minifier
I suggest to remove everything about "testplane-image-minifier" as it is deprecated and no longer needed since `testplane` started using sharp

Reply:
No, I didn't rename anything. There are no mentions that this plugin is deprecated...

I suggest to discuss this in the next iteration. Added this to my TODO list.